pkg/store/storepb/testutil/series.go

```diff
@@ -256,20 +308,22 @@ func TestServerSeries(t testutil.TB, store storepb.StoreServer, cases ...*Series
 					}
 
 					// Huge responses can produce unreadable diffs - make it more human readable.
-					if len(c.ExpectedSeries) > 4 {
+					if len(c.ExpectedSeries) > 1 {
 						for j := range c.ExpectedSeries {
 							testutil.Equals(t, c.ExpectedSeries[j].Labels, srv.SeriesSet[j].Labels, "%v series chunks mismatch", j)
 
 							// Check chunks when it is not a skip chunk query
 							if !c.Req.SkipChunks {
 								if len(c.ExpectedSeries[j].Chunks) > 20 {
-									testutil.Equals(t, len(c.ExpectedSeries[j].Chunks), len(srv.SeriesSet[j].Chunks), "%v series chunks number mismatch", j)
+									require.Equal(t, len(c.ExpectedSeries[j].Chunks), len(srv.SeriesSet[j].Chunks), "%v series chunks number mismatch", j)
```

Comment:
Why the change here? :thinking:

Reply:
I made most of these changes to troubleshoot test failures since `require` produces a nicer diff. They should all be reverted now.